src/home/profile/Settings.js

```diff
@@ -67,6 +71,12 @@ export default class Settings extends React.Component {
                     this.props.navigation.goBack()
                 });
             }
+            if (address !== originalProfile.address) {
```

Comment:
This is already being done on lines 67-73, if you are going to do something similar with email make sure to do it in a similar fashion so that the previous page auto updates and the screen goes back at the right time

Reply:
I forgot to remove this when i merged. I also removed the rest of the email stuff.